coffee-shop/src/pages/index.tsx

```diff
@@ -0,0 +1,14 @@
+import Head from 'next/head'
+import styles from '../styles/Home.module.css'
+//import Navigation from '../components/Navigation/navigation'
+import Logo from '../components/Logo/logo'
+
+export default function Home() {
+  return (
+    <div className={styles.container}>
+
+      <Logo />
```

Comment:
You should pass props into component:
ex: `<Logo src="/images/logo.png"` alt="logo" width={201} height={86} />`